docs/alerts/monitors/settings.md

```diff
@@ -115,6 +117,21 @@ To add a tag while creating a new monitor:
    * Apply an existing tag by selecting the key and its possible values from the dropdown.
 1. Click **Save**.
 
+
+### What happens next?
+
+Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.
+
+To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->
```

Comment:
Also users cannot edit the tags on the Alerts. They can only see it and use them for filtering.

Although users can edit tags on the Monitor, the edited tags will not be updated on the alerts that were previously triggered. We should mention this too.